### PR TITLE
created backend student model

### PR DESCRIPTION
**Pull request by tinydinosaurs**
created new file for backend student model

### CODE REVIEW

File: models/Student.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require('./StoryRead');
+module.exports = bookshelf.model('Student', {
+	tableName: 'students',
+	hasTimestamps: ['createdAt', 'updatedAt', 'deletedAt'],
+	authentication: function() {
```
Comment (alarner) [inline models/Student.js]: 👎 I think this should be changed from `authentication:...` to `storiesRead:`... or something like that

**Comment by willcook4code** [comment]
👍

File: models/Student.js (hunk 1 above)
Comment (alarner) [inline models/Student.js]: Because the student model is not really related to the authentication model

Comment (tinydinosaurs) [inline models/Student.js]: whoops! that's what I get for copying and pasting.

**Comment by tinydinosaurs** [comment]
fixed issue @alarner noted

**Comment by NJHawthorne** [comment]
👍

**Comment by hammour** [comment]
👍

**Pull request merged**